js/SwordDepositPointsFormHandler.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file js/SwordDepositPointsFormHandler.js
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @package plugins.generic.sword
+ * @class SwordDepositPointsFormHandler
+ *
+ * @brief SWORD plugin deposit points form handler.
+ */
+(function($) {
```

Comment:
Did you run your JS through the linter?

Reply:
No. for OJS I usually rely on Travis tests. Any other way to do it locally from the codebase?

Reply:
This is a separate repo so the Travis tests won't run on it. On my machine I can run

    bash lib/pkp/tools/buildjs.sh

but I'm not sure that a) this tests content in the plugin area, or b) that it won't take some local setup first (it's been a while).